### PR TITLE
Bugfix/11 fixed dockerfile and provided update to solutions

### PR DESCRIPTION
**Pull request by dnLeiting**
See: https://github.com/smart-data-lake/getting-started/issues/11

Updates:
    Update the provided solutions with the newest version of the SDLB API
    Update the Dockerfile to curl the latest minor version available under https://dlcdn.apache.org/spark/

### CODE REVIEW

File: config/application.conf.part-2-solution [hunk 1]
```diff
@@ -20,7 +20,7 @@ dataObjects {
   }
   stg-departures {
     type = JsonFileDataObject
-    path = "~{id}"
+    path = stg-departures
```
Comment (mand35) [inline config/application.conf.part-2-solution:23]: Why we cannont use "~{id}" here?

Comment (dnLeiting) [reply config/application.conf.part-2-solution:23]: I took the config from the link provided in the SDLB getting started documentation.
https://smartdatalake.ch/assets/files/application-part2-historical-62b47cb172772c79ba8f08c7082ca360.conf

Thus both solutions are the same. In the solution provided with the link no "~{id}" is used.
What do you think, should i still use the placeholder "~{id}" in the conf/application.conf.part-2-solution file?

Comment (mand35) [reply config/application.conf.part-2-solution:23]: good question. There is consistency against presenting different ways to specify. Maybe we describe it a little better and choose:
```
    path = stg-departures     # can be replaced by the DataObject ID by using "~{id}"
```
and thus we describe it once in the code in the first element. How does that sound for you?

**Review by mand35 — APPROVED** [review]
great catches. Many thanks.

**Pull request merged**